### DISCUSSION TITLE
urllib3 and chardet doesn't match a supported

### DISCUSSION THREAD

**Discussion by setiawan-chandra** [MHProDev/MHDDoS#533] [Q&A]
<img width="598" alt="Screenshot 2024-03-03 075503" src="https://github.com/MatrixTM/MHDDoS/assets/107093709/320b4cb0-62b8-413c-8875-53d0eabdecec">
urllib3 and chardet doesn't match a supported how i can fix it?

**Comment by Lefoxxxx** [comment]
Try to resolve using this code
`pip install --upgrade urllib3 chardet charset_normalizer requests`

That will update `urllib3`, `chardet`, `charset_normalizer`, and `requests`

Hope this helps!
